authenticate/handlers.go

```diff
@@ -451,12 +451,18 @@ func (a *Authenticate) userInfo(w http.ResponseWriter, r *http.Request) error {
 		s.ID = uuid.New().String()
 	}
 
+	isImpersonated := false
 	pbSession, err := session.Get(ctx, state.dataBrokerClient, s.ID)
```

Comment:
This error looks unchecked? What happens if `err!=nil` here?

Reply:
e.g. could `pbSession` be `nil` and cause a nil pointer? Probably worth adding a test.

Reply:
no it can't. `Get` methods in protobuf are nil-safe. But sure I can change the code.

Reply:
If the answer is no we don't need to update the code.